app/views/admin/secrets/fail.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Secret Backend is Down!!</h1>
```

Comment:
we don't really know if it's down or if something else weird happened ... `Error reading secrets: <error>`